### PR TITLE
Add HTML Keyboard Slider Response

### PR DESCRIPTION
**Pull request by Max-Lovell**
Adds a slider that accepts keypresses and a bunch of new parameters (maybe too many...). Updated Jest config to click continue button, and a few other things in there that can be commented out to check other features. A few examples are in the appropriate folder, too - all parameters listed. Let me know if clarifications needed.

### CODE REVIEW

**Comment by jodeleeuw** [comment]
Thanks @Max-Lovell!

Would you be willing to fill out the documentation with the parameters?

**Comment by jodeleeuw** [comment]
It'll also need a changeset

`npm run changeset` in the root directory.

**Comment by Max-Lovell** [comment]
Hi Josh, thanks for the quick feedback - happy to add descriptions, although I thought these were created automatically? If so, perhaps I've screwed up the markdown but can't see where - can you point me in the right direction?

**Comment by jodeleeuw** [comment]
Ah, we're moving in that direction, but it isn't quite done yet. You're welcome to leave as is, but it might be a couple of weeks before we have this all set up.

**Comment by Max-Lovell** [comment]
Hi Josh, Ah okay then - all done! Thanks

**Comment by Max-Lovell** [comment]
Note, implements suggestions from:
https://github.com/jspsych/jsPsych/issues/2570
https://github.com/jspsych/jsPsych/pull/2827

**Comment by jodeleeuw** [comment]
One last thing - can you update the table in the main README for the repo to include the plugin? (hoping to automate this step in the future...)

**Comment by Max-Lovell** [comment]
Sure - I actually forgot to do that last time. Listed my last contrib 'survey-slider' to the main list aswell, hopefully that's alright under this PR?

File: README.md
Comment (jodeleeuw) [inline README.md:1]: Can you put these in their alphabetical order spot?

**Review by jodeleeuw — CHANGES_REQUESTED** [review]

File: README.md
Comment (Max-Lovell) [reply README.md:1]: Done 👍

**Comment by Max-Lovell** [comment]
Sorry, could you briefly reopen? I've noticed I put the wrong link in the list of packages: 
`https://github.com/jspsych/jspsych-contrib/blob/main/packages/plugin-html-keyboard-slider/docs/jspsych-html-keyboard-slider.md`
should be:
`https://github.com/jspsych/jspsych-contrib/blob/main/packages/plugin-html-keyboard-slider/docs/html-keyboard-slider.md`

**Pull request merged**